### DISCUSSION TITLE
Set custom models folder on initial(first) launch

### DISCUSSION THREAD

**Discussion by davehornik** [vladmandic/sdnext#513] [Ideas]
Let's say there's someone like me trying this repo and I have all the models in AUTOMATICs repo folder.
I was wondering if it's possible to add prompt asking if there's a _models_ folder already?

To prevent wasting space/time downloading again the default pruned sd1.5.
_(I've got 1gig dl speed and free hdd for this, but some folks are not that lucky)_
I guess there's a lot people coming over from A1111 to test this.
It is not that big of a deal to copy/move it over. It's more like QoL thing that can help some people 🤷‍♂️

**Comment by vladmandic** [comment]
you have two options:
- use `--ckpt` to load specific checkpoint and then from UI set model paths to be same as in A1111
- use `--model-dir` to point `/models` to be same as in A1111

**Reply by matbeedotcom** [reply]
command is `--models-dir` for any readers copying and pasting

**Comment by Soulreaver90** [comment]
I’ve been reinstalling the repo for testing so I have an automated process. I’m on Linux so I delete the models folder that gets created and create a symbolic link of the auto1111 models folder to vlad’s folder. `ln -s source destination`
The same can be done on windows, I use it to point some of my folders to my second larger drive for storage.
